### PR TITLE
oh no, a bug!

### PR DESCRIPTION
**Pull request by waltaskew**

### CODE REVIEW

File: usl_pipeline/usl_pipeline/adder.py [hunk 1]
```diff
@@ -1,2 +1,6 @@
 def add(x, y):
     return x + y
+
+
+def sub(x, y):
```
Comment (waltaskew) [inline usl_pipeline/usl_pipeline/adder.py:5]: I think you could add some documentation here

File: usl_pipeline/usl_pipeline/adder.py [hunk 2]
```diff
@@ -1,2 +1,6 @@
...
+    return x - y
```
Comment (waltaskew) [inline usl_pipeline/usl_pipeline/adder.py:6]: great job with the subtraction! what a function!

Comment (waltaskew) [reply usl_pipeline/usl_pipeline/adder.py:6]: thanks :)

File: usl_pipeline/usl_pipeline/adder.py (hunk 1 above)
Comment (waltaskew) [reply usl_pipeline/usl_pipeline/adder.py:5]: maybe later...

**Comment by ahmustafa** [comment]
good to go

**Comment by ahmustafa** [comment]
good

**Pull request merged**